### DISCUSSION TITLE
面積量測與垂直距離的尺寸不同

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#514] [Q&A]
您好,
請問3D建模出來的物件上
面積量測與垂直距離量測的結果相差很大
這是什麼問題導致?
謝謝
下圖左邊是面積量測,右邊是垂直距離量測
![2024-02-17_114639](https://github.com/zouyaoji/vue-cesium/assets/5283322/a33784a9-06e9-4d57-b5c7-214fc936fab4)

**Comment by stan88** [comment]
是不是geodesic.surfaceDistance的問題
只算到X,Y的距離

**Reply by zouyaoji** [reply]
垂直测量的终点点没在建筑上的，起点是拾取位置，终点与起点的线是一个垂直地表的线。
